velox/vector/DictionaryVector.h

```diff
@@ -186,6 +186,7 @@ class DictionaryVector : public SimpleVector<T> {
   }
 
   std::string toString(vector_size_t index) const override {
+    loadedVector(); // load lazy vector if needed
```

Comment:
nit: Please move the comment to a separate line above the code and capitalize the first character and add a period at the end (because comments are supposed to be sentences). 

The change makes sense to me. @bikramSingh91, could you take a look too? Thanks!

Reply:
@peterenescu while this would work, it will have the side effect of loading the vector which can be a cost when used in production code or can change the conditions of a unit test rendering it ineffective. In that case, we would have to make sure that any test that might rely on these conditions retains them if it is also using toString() in any capacity (like in ExpressionVerifier).
I discussed this offline with Wei, and we agreed that it would be better to return an exception if toString() is called on an unloaded vector. Since the primary use case is to use toString() for debugging, this would make it obvious to the user what is happening here instead of silently loading the vector.